_raml/apps/products/products.raml

```diff
@@ -146,6 +146,18 @@ post:
     responses:
       204:
 
+  /availabilityNotifications:
+    displayName: Product Availability Notification
+    post:
+      description: Modifies information on the email notification assignments of a single product by adding the email address of the customer and the storefront locale. *epagesNow
```

Comment:
```suggestion
      description: Enables the availability notification for the given product and customer email address. The language of the notification is based on the locale of the customer's storefront. *epagesNow
```
I would phrase the description differently because in my opinion "modifies information on the email notification assignment" is a bit abstract. Let me know what you think.